src/doc/src/reference/profiles.md

```diff
@@ -171,7 +171,8 @@ The valid options are:
   similar to "fat".
 * `"off"`: Disables LTO.
 
-See also the [`-C linker-plugin-lto`] `rustc` flag for cross-language LTO.
+When enabled this setting can also potentially pass the [`-C linker-plugin-lto`]
+`rustc` flag for cross-language LTO.
```

Comment:
This sentence is more trying to provide a breadcrumb to the user for more information on *how* to do cross-language LTO. What do you think about something like the following:

```suggestion
See the [linker-plugin-lto chapter] if you are interested in cross-language LTO. This is not yet supported natively in Cargo, but can be performed via RUSTFLAGS.

[linker-plugin-lto chapter]: ../../rustc/linker-plugin-lto.html
```

And also update the first sentence of this section to something like:

> The `lto` setting controls rustc's [`-C lto`], [`-C linker-plugin-lto`], and [`-C embed-bitcode`] options, which controls LLVM's ...

(with links to each of those options).

Reply:
Thanks for the edits, updated!